frontend/src/main/pages/Drivers/DriverAvailabilityIndexPageAdmin.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useBackend } from 'main/utils/useBackend';
+
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import DriverAvailabilityTable from 'main/components/Driver/DriverAvailabilityTable';
+import { useCurrentUser } from 'main/utils/currentUser'
+// import { Button } from 'react-bootstrap';
+
+export default function DriverAvailabilityIndexPage() {
+
+    const currentUser = useCurrentUser();
+
+    const { data: availabilities, error: _error, status: _status } =
+        useBackend(
+            // Stryker disable all : hard to test for query caching
+            ["/api/driverAvailability"],
+            { method: "GET", url: "/api/driverAvailability/admin/all" },
+            []
+            // Stryker restore all 
+        );
+
+    // const createButton = () => {
+    //     if (hasRole(currentUser, "ROLE_ADMIN")) {
+    //         return (
+    //             <Button
+    //                 variant="primary"
+    //                 href="/driverAvailability/create"
+    //                 style={{ float: "right" }}
+    //             >
+    //                 Create Driver Availability
+    //             </Button>
+    //         )
+    //     } 
+    // }
```

Comment:
Remove commented code, other than that its good

Reply:
resolved!